source/draw/DrawState.ooc

```diff
@@ -29,6 +29,8 @@ DrawState: cover {
 	opacity := 1.0f
 	_transformNormalized := FloatTransform3D identity
 	viewport := IntBox2D new(0, 0, 0, 0)
+	destination := IntBox2D new(0, 0, 0, 0) // TODO: Accept FloatBox2D
```

Comment:
I would get one less argument if I give in local coordinates but floating. Someone might however mistake FloatBox2D for normalized coordinates then. If I store in normalized coordinates, I have to send in the size of the image or take it from the DrawState but then I can have (0, 0, 1, 1) as default value instead of checking if the area exists.